ArduPlane/AP_ExternalControl_Plane.cpp

```diff
@@ -19,4 +19,34 @@ bool AP_ExternalControl_Plane::set_global_position(const Location& loc)
     return plane.set_target_location(loc);
 }
 
+/*
+  Sets the target airspeed.
+*/
+bool AP_ExternalControl_Plane::set_airspeed(const float airspeed)
```

Comment:
This new method could be used in `MAV_RESULT GCS_MAVLINK_Plane::handle_command_int_guided_slew_commands` ?

Reply:
Correct! That was the idea with the external control library - we start piping GCS commands through it soon.

Reply:
Done!